frontend/src/components/Form/PrioritySetting.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { FC } from "react";
+import styled from "styled-components";
+
+interface iPriorityProps {
+  priority: number[];
+  onHandleFormElement: React.ChangeEventHandler<HTMLInputElement>;
+}
+
+const PrioritySetting: FC<iPriorityProps> = ({
+  priority,
+  onHandleFormElement,
+}) => {
+  return (
+    <>
+      <LabelText>Priority</LabelText>
+
+      <RadioGroup>
+        {priority.length > 0 &&
+          priority.map((radioItem) => (
+            <RadioItem key={radioItem}>
+              <RadioInput
+                type="radio"
+                name="priority"
+                value={radioItem}
+                id={`radio-item-${radioItem}`}
+                onChange={onHandleFormElement}
+              />
+
+              <FormCheckRadio htmlFor={`radio-item-${radioItem}`}>
+                {radioItem === 1 ? `high` : radioItem === 2 ? `medium` : `low`}
```

Comment:
I'm not a big fan of using the ternary expression like this.
It should be kept simple like this: 
```
 condition === true ? 'true value' : 'false value'
```

When you need it to be a bit more complex, extract it to so separate function


e.g:
```
const setRadioPriority = (radioPrio: number) => {
 // there are obviously multiple ways, but mapping everything to an object is one:
  const prioList = {
    1: 'high',
    2: 'medium',
    3: 'low'
 }

return prioList[radioPrio]
}
```

and your JSX would be:

```
<FormCheckRadio htmlFor={`radio-item-${radioItem}`}>
       { setRadioPriority(radioItem) }
</FormCheckRadio>
````

Reply:
Or third option
```javascript
const PrioritySetting: FC<iPriorityProps> = ({
  priority,
  onHandleFormElement,
}) => {
  const prioList = ['high', 'medium', 'low']
  return (
    <>
       // priolist is available here

```

Reply:
is there a downfall for using the ternary like this or is this a personal preference?

Reply:
The downfall is that the code is harder to read and therefor you're introducing unnecessary complexity.
Your code must always be as readable as possible; for everybody that reads your code: your colleagues, someone who just wants to look at your code, your future self